.gitignore

```diff
@@ -12,3 +12,4 @@
 # Misc
 .env*
 *.log*
+.DS_Store
```

Comment:
```suggestion
```

One of my personal hatreds: polluting `.gitignore` with IDE/OS-specific things which belong in `$XDG_CONFIG_HOME/git/ignore` instead[^gitignore]

[^gitignore]: https://git-scm.com/docs/gitignore

Reply:
This is better in principle I agree.

But I wonder being an open source project, that may have untrusted/unknown contributors overtime, still worth paying the price for sake of enforcing rules via project scoped .gitignore

ie. It save us from a manual check/review comment in the future when will inevitably happen